roles/helm_common/defaults/main.yml

```diff
@@ -0,0 +1 @@
+bin_dir: /usr/local/bin
```

Comment:
This is already set elsewhere, no?

Reply:
Yes, but considering that:
- bin_dir is misnamed (bin_dir="/usr/bin" ?) and the name of the variable could change
- /usr/local/bin/ is the preferable destination to install helm without package
- we will detect rapidly if helm has been changed of location
- kubespray-defaults is slow

I find reasonnable to have the duplication of the variable

Reply:
But I don't care, so I could change I you want (to be more opensource friendly or whatever)